src/backend/distributed/commands/index.c

```diff
@@ -556,16 +561,26 @@ PreprocessReindexStmt(Node *node, const char *reindexCommand,
 			state.locked_table_oid = InvalidOid;
 
 			Oid indOid = RangeVarGetRelidExtended(reindexStatement->relation,
-												  lockmode, 0,
+												  lockmode, RVR_MISSING_OK,
 												  RangeVarCallbackForReindexIndex,
 												  &state);
+			if (!OidIsValid(indOid))
+			{
+				return NIL;
```

Comment:
again, lets add 1 line comments like `/* let Postgres decide on the behavior and the error message */`